sympy/geometry/tests/test_ellipse.py

```diff
@@ -470,7 +470,25 @@ def test_is_tangent():
     assert e1.is_tangent(Polygon((-3, -5), (-3, 5), (3, 5), (3, -5))) is True
     assert c1.is_tangent(Polygon((-3, -5), (-3, 5), (3, 5), (3, -5))) is False
     assert e1.is_tangent(Polygon((0, 0), (3, 0), (7, 7), (0, 5))) is False
-    assert e1.is_tangent(Polygon((3, 12), (3, -12), (6, 5))) is True
+    assert e1.is_tangent(Polygon((3, 12), (3, -12), (6, 5))) is False
+    assert e1.is_tangent(Polygon((3, 12), (3, -12), (0, -5), (0, 5))) is False
```

Comment:
I removed some tests.
if a rectangle is tangent to a circle on one side, but the opposite side passes through the circle, then the opposite side must intersect the circle at two points (since the line is not tangent to the circle on that side)